nobl9/resource_agent.go

```diff
@@ -115,6 +116,19 @@ func agentSchema() map[string]*schema.Schema {
 	return agentSchema
 }
 
+//nolint:unparam
```

Comment:
instead of adding this nolint you can just use `_` for parameter name

Reply:
Done. Also removed a few other unnecessary ones. Thanks!